src/client.ts

```diff
@@ -943,6 +943,8 @@ class Client {
    *
    * Note that the `identity` input takes snake_case values.
    *
+   * registerIdentity() is intended for internal use. The JS SDK (@toznysecure/sdk/node) should be used instead to register Identities.
+   *
```

Comment:
please add: what are the limitations when using this method, and a brief explainer on why (e.g. no notes are written for the broker to be able to do password recovery/reset)